docs/all.md

```diff
@@ -1039,14 +1040,15 @@ Any changes to policies and procedures defined both here and in the SSPP must be
 that the security of the system is maintained.
 
 1. [Alerts, Notifications, Monitoring](#alerts)
-2. [Restaging Apps](#restaging-apps)
-3. [Smoke-testing the App](#smoke-testing)
-4. [Simulated bulk send testing](#simulated-bulk-send-testing)
-5. [Configuration Management](#cm)
-6. [DNS Changes](#dns)
-7. [Known Gotchas](#gotcha)
-8. [User Account Management](#ac)
-9. [SMS Phone Number Management](#phone-numbers)
+1. [Restaging Apps](#restaging-apps)
+1. [Deploying to Production](#deploying-to-production)
+1. [Smoke-testing the App](#smoke-testing)
+1. [Simulated bulk send testing](#simulated-bulk-send-testing)
+1. [Configuration Management](#cm)
+1. [DNS Changes](#dns)
+1. [Known Gotchas](#gotcha)
```

Comment:
why all 1s?

Reply:
Good question!  I'm taking advantage of Markdown's formatting of numbered lists here, so that whenever we change the order or add another item, we don't have to readjust the whole entire numbering too - Markdown figures it out for us. 🙂

You can see this with other lists in the documentation as well!